flood_risk_engine.gemspec

```diff
@@ -1,4 +1,4 @@
-$:.push File.expand_path("../lib", __FILE__)
+$LOAD_PATH.push File.expand_path("../lib", __FILE__)
```

Comment:
New version is clearer to me.

Also this wasn't picked up previously because gemspec files were excluded. I don't think they should be.